### DISCUSSION TITLE
Week 6- Instructional approaches and Final challenges

### DISCUSSION THREAD

**Discussion by CJ3315** [USU-ITLS-x265-23-summer/class-repo#128] [General]
Thoughts and Reflection on Materials: 
I was able to take a lot away from this week's materials. Throughout my other courses, I have had my eyes opened to the world of accessibility and how important this topic is when it comes to any type of design. The main thing that I took away from the video and readings was how important it is to get feedback from others when creating something. It is so hard for us to really relate to others if we do not have the same struggles. 
In past classes, I have done more research on accessibility tools, and one front that I think this topic is really awesome is in video games. Here is an example of these tools being implemented: 
https://youtu.be/en9Mic3eQ5Y

Challenges for Proposed final projects: 

- Portfolio 
For this portfolio project, I want to utilize my past experience in Dreamweaver. This has been where the bulk of my work with HTML has been. I do anticipate issues with functionality and the appearance of the design as I am still a novice with this software. 

- Online Game
I have built many games in the past through scratch coding, however this has been through block coding. I would have to gain more practice through JavaScript in order to fully execute this idea.

**Comment by jakegros** [comment]
@CJ3315, I have been going back and forth between a website or creating a game for my project.  However, I realize that either way, I want to get into coding basic games and go from there.  As I have explored some resources for some guidance, I came across this source that seems to have good info that helps me understand a way to go about it. 
https://www.freecodecamp.org/news/how-to-code-a-simple-game/

**Comment by metasean** [comment]
For multiple reasons, I strongly advise against using Dreamweaver.

It's been a while since I used Dreamweaver, but when I last did, it added a whole lot of "cruft" that actually made it really hard to maintain or to really customize.  

Remember, I'm grading the quality of the html (as you were supposed to be learning that up to this point), and the likelihood that Dreamweaver code will meet my expectations are on the slim side, because of the aforementioned cruft.

Also, you will need to keep the project code in GitHub up-to-date, and to do so, you'll need to install and learn to use git from the command line.
